odiglet/pkg/metrics/metrics.go

```diff
@@ -0,0 +1,169 @@
+package metrics
+
+import (
+	"context"
+
+	odigosv1 "github.com/odigos-io/odigos/api/odigos/v1alpha1"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	otelmetric "go.opentelemetry.io/otel/metric"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	otelMeterName   = "github.com/odigos.io/odigos/odiglet"
+	languageAttrKey = "language"
```

Comment:
consider using semantic attribute for this, you can checkout [telemetry.sdk.language](https://opentelemetry.io/docs/specs/semconv/registry/attributes/telemetry/)